blitzstudios/sleeper-mini-core
# Upgrading react native to 72 and repack to 3.6.

### Description

This commit upgrades react native to 72.7 and @callstack/repack to 3.6. This will coincide with the same library upgrades in the main sleeper app, and will be mandatory for all mini developers to integrate in order for their minis to continue appearing in the app. 

(Notice of the upgrade timeline will be sent out soon.)

### How To Test

1. Connect your mini project with a react native 72 version of the sleeper app.
2. Ensure all functionality still works.
